### PR TITLE
Allow being provided a list of services to bound to the apps

### PR DESCRIPTION
**Pull request by l-lafin**
I'm adding a variable to be possible to provide a list of services to be bound to the apps. Such a feature is needed for example to send the apps logs to the log bucket.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -141,3 +141,27 @@ variable "alertmanagers_endpoints" {
...
   default     = []
 }
+
+variable "thanos_service_bindings" {
+  type        = list(object({ service_instance = string }))
+  description = "A list of service instances that should be bound to the thanos app"
+  default     = []
+}
+
+variable "thanos_query_service_bindings" {
```
Comment (jdelucaa) [inline variables.tf:151]: duplicated

**Review by jdelucaa — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (l-lafin) [reply variables.tf:151]: Removed 😃

**Comment by l-lafin** [comment]
Hi @loafoe, could you have a look at this PR? I'm adding a configuration to allow us to connect a logging service to the apps.

**Comment by loafoe** [comment]
@l-lafin merged and created a new release

**Pull request merged**